Lesson_11/Observer/Realtor.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Lesson_11
+{
+    internal class Realtor : ISubscriber
+    {
+        public int ExpectedPrice;
+
+        public Realtor(int expectedPrice)
+        {
+            ExpectedPrice = expectedPrice;
+        }
+
+        public void Update(Object obj)
+        {
+            Flat flat = (Flat)obj;
```

Comment:
instead of object better to send exactly flat or interface lPricable or any other option, object it's not thee best option because you use resources for boxing unboxing